src/components/BreadCrumb/BreadCrumb.vue

```diff
@@ -107,6 +112,10 @@ export default class BreadCrumb extends Vue {
   cursor: pointer !important;
 }
 
+.inactive-crumb {
+  cursor: default !important;
+}
```

Comment:
1. I think `!important` can still be overridden by [more specific CSS](https://www.w3schools.com/css/css_specificity.asp). In some cases I have had to use cascading selectors to override something ([example](https://github.com/severinbeauvais/business-filings-ui/blob/8991a1fdabdb37e57bc6205b020d10d1809147df/src/components/dialogs/StaffPaymentDialog.vue#L106)).

2. Would [pointer events](https://www.w3schools.com/cssref/css3_pr_pointer-events.asp) be another, or alternate, property that meets your need?

Reply:
1: In this instance it's working as expected. 
2: I believe it would do the same thing yes.

Reply:
OK, perfect.